.github/workflows/find-inactive-collaborators.yml

```
@@ -42,9 +42,15 @@ jobs:
           author: Node.js GitHub Bot <github-bot@iojs.org>
           branch: actions/inactive-collaborators
           body: |
-            This PR was generated by tools/find-inactive-collaborators.yml.
+            This pull request was automatically generated by [`.github/workflows/find-inactive-collaborators.yml`](https://github.com/nodejs/node/blob/HEAD/.github/workflows/find-inactive-collaborators.yml).
+
+            The following collaborators have been inactive since ${{ steps.find-inactive-collaborators.outputs.SINCE }}, and are being moved to emeritus status:
+
+            ${{ steps.find-inactive-collaborators.outputs.INACTIVE_COLLABORATORS }}
+
+            ---
 
-            @nodejs/tsc Please follow up with the [offboarding tasks](https://github.com/nodejs/node/blob/main/doc/contributing/offboarding.md).
+            @nodejs/tsc, please review the [offboarding tasks](https://github.com/nodejs/node/blob/main/doc/contributing/offboarding.md).
```

This looks unrelated
```suggestion
            @nodejs/tsc Please follow up with the [offboarding tasks](https://github.com/nodejs/node/blob/main/doc/contributing/offboarding.md).
```